Bottom: 3abaffadea7dee3735852fe57b6e9d533f680afb
Top:    63e2129fa90d65f287b2c3ee706a130d1da7c34f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 22:44:07 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
